README.md

```diff
@@ -282,7 +283,7 @@ If there is no data to send for a given parameter, the parameter should not be i
 
 | Tag Name | Type | Introduced | Description |
 |----------|------|------------|-------------|
-| reason | string | 5.1.0 | Specify a string describing the reason of failure |
+| reason | String | 5.1.0 | Specify a string describing the reason of failure |
```

Comment:
```suggestion
| reason | String | 5.1.0 | A string describing the reason of failure |
```

Reply:
Fixed in https://github.com/smartdevicelink/protocol_spec/pull/34/commits/00a63ae3f83fa8097e36d9c6ffc5fe3cbe78401e